SCClassLibrary/Common/Geometry/Rect.sc

```diff
@@ -144,6 +144,7 @@ Rect {
 	}
 
 	asRect { ^this }
+	asSize { ^Size(width, height) }
```

Comment:
Note: `Rect` has a `-size` method, so this is a synonym to that method. But this brings `Rect` into line with a number of other related classes which respond to `-asSize`: `Number`, `Point`, and `Size`. So methods can accept all of these types when converting to a `Size`, and it's more explicit (`size` in SC is also the length of an array).

Reply:
yes, that makes sense.